build.gradle.kts

```diff
@@ -6,7 +6,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath("com.android.tools.build:gradle:${Versions.androidGradlePlugin}")
+        classpath("com.android.tools.build:gradle:4.2.2")
```

Comment:
This should be rolled back to the old line 
```gradle
classpath("com.android.tools.build:gradle:${Versions.androidGradlePlugin}")
```

See PR message for explanations (we'll upgrade the plugin in a different commit).